### ISSUE TITLE
Unable to access settings while using VoiceOver

### ISSUE THREAD

**Issue by tspivey** [tailscale/tailscale#9611]
### What is the issue?

I can't easily access the button that lets me access settings/log out with VoiceOver.
I would expect this to show up somewhere on the screen, but it doesn't.
The only way I can get to it is with screen recognition.

### Steps to reproduce

1. With VoiceOver active, start at the top of the screen with the connected switch.
2. Flick right to explore the interface.
While doing this, notice that nothing appears to let you adjust settings/log out.
3. Turn on screen recognition and repeat the process. The button for this will be there somewhere, though I can't get it to show up consistently.

### OS

iOS

### OS version

16.6.1

### Tailscale version

1.50.0

### Bug report

BUG-cd2ef827aad7f1910be814efec7506150e604cf52e6845aa16440c1b46e76735-20231001085623Z-5575f7b0b516ff9d
